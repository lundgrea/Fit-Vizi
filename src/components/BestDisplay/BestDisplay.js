import React from 'react';

export const BestDisplay = (props) => {
  return (
    <>
      <h2>Best Average Power Outputs</h2>
      <h4>1 Minute: {props.data.topOne.averagePower}</h4>
      <p>Record Began {props.data.topOne.startIndex} Seconds Into and Lasted 60 Seconds </p>
      <h4>5 Minutes: {props.data.topFive.averagePower}</h4>
      <p>Record Began {props.data.topFive.startIndex} Seconds Into and Lasted 5 Minutes</p>
      <h4>10 Minutes: {props.data.topTen.averagePower}</h4>
      <p>Record Began {props.data.topTen.startIndex} Seconds Into and Lasted 10 Minutes</p>
      <h4>15 Minutes: {props.data.topFifteen.averagePower}</h4>
      <p>Record Began {props.data.topFifteen.startIndex} Seconds Into and Lasted 15 Minutes</p>
      <h4>20 Minutes: {props.data.topTwenty.averagePower}</h4>
      <p>Record Began {props.data.topTwenty.startIndex} Seconds Into and Lasted 20 Minutes</p>
    </>
  )
}

export default BestDisplay