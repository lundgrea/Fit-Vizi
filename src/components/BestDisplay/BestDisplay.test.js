

import React from 'react';
import { shallow } from 'enzyme';
import { BestDisplay } from './BestDisplay';

describe('BestDisplay', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<BestDisplay 
      topOne={'averagePower'}
      topFive={'averagePower'} 
      topTen={'averagePower'} 
      topFifteen={'averagePower'} 
      topTwenty={'averagePower'}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});


