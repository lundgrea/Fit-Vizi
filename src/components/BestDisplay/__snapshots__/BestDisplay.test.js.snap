// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BestDisplay should match the snapshot with all data passed in correctly 1`] = `
<Fragment>
  <h2>
    Best Average Power Outputs
  </h2>
  <h4>
    1 Minute: 
  </h4>
  <p>
    Record Began 
     Seconds Into and Lasted 60 Seconds 
  </p>
  <h4>
    5 Minutes: 
  </h4>
  <p>
    Record Began 
     Seconds Into and Lasted 5 Minutes
  </p>
  <h4>
    10 Minutes: 
  </h4>
  <p>
    Record Began 
     Seconds Into and Lasted 10 Minutes
  </p>
  <h4>
    15 Minutes: 
  </h4>
  <p>
    Record Began 
     Seconds Into and Lasted 15 Minutes
  </p>
  <h4>
    20 Minutes: 
  </h4>
  <p>
    Record Began 
     Seconds Into and Lasted 20 Minutes
  </p>
</Fragment>
`;
